### DISCUSSION TITLE
Welcome to outerspace-blockchain Discussions!

### DISCUSSION THREAD

**Discussion by JesusGautamah** [JesusGautamah/outerspace-blockchain#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aftabtayyaba** [comment]
can we add payment module only using blockchain technology in any project?

**Reply by JesusGautamah** [reply]
This is a blockchain concept in a web based server, it will have own transactions API, but will not have payment API like Stripe included, so it will be possible to do payments with coin based on your chain(or my when the beta version release). So if the coin created by the chain, has self value, yeah, it will be possible using the transactions API and your wallet, maybe it will be added a method to made seller transactions securely and easy to setup in other projects.

It is currently in unstable alpha version, and is not supposed to use in production.
NFTs generators and a Redis NoSQL management is coming to project and i think this project can be more interesting later.
Sorry for my bad english.
Thanks for your support!
